Datasets/data-connect-dataset-onedrivesynchealth.md

```diff
@@ -13,6 +13,13 @@ ms.custom: datasets:dataset-name
 
 The OneDrive Sync Health dataset includes information on devices running OneDrive for Business. This includes one object for every report-enabled OneDrive Sync client in the tenant. Reports include Sync app information, user details, error details, and relevant timestamps.
 
+NOTE:
```

Comment:
Please remove changes from this file as it is a tenant shard dataset

Reply:
updated